modules/statics/src/networks.ts

```diff
@@ -963,6 +963,10 @@ class ZkSyncTestnet extends Testnet implements EthereumNetwork {
   chainId = 300;
   nativeCoinOperationHashPrefix = '300';
   tokenOperationHashPrefix = '300-ERC20';
+  forwarderFactoryAddress = '0xdD498702F44c4da08EB9E08D3F015eEfE5cB71Fc';
+  forwarderImplementationAddress = '0xBE69CAE311191Fb45E648ed20847F06faD2DBAB4';
+  walletFactoryAddress = '0x4550e1e7616D3364877FC6c9324938Dab678621a';
+  walletImplementationAddress = '0x92dB2759d1DcA129a0D9d46877f361Be819184C4';
 }
```

Comment:
Can you please make these addresses all small case

Reply:
updated !!